cmd/kube-apiserver/app/aggregator.go

```diff
@@ -134,9 +135,11 @@ func createAggregatorServer(aggregatorConfig *aggregatorapiserver.Config, delega
 	crdRegistrationController := crdregistration.NewAutoRegistrationController(
 		apiExtensionInformers.Apiextensions().InternalVersion().CustomResourceDefinitions(),
 		autoRegistrationController)
+	openapiController := openapi.NewController(apiExtensionInformers.Apiextensions().InternalVersion().CustomResourceDefinitions())
```

Comment:
```
// instantiate apiextensions openapi controller here because it needs direct access to the openapi aggregator
// TODO: add a more direct aggregation trigger channel through the delegation chain
```